Blish HUD/GameServices/Modules/UI/Views/ManagePkgView.cs

```diff
@@ -0,0 +1,220 @@
+        }
+
+        private ModuleContributor _moduleContributor;
+
+        public ModuleContributor ModuleContributor {
+            get =>
+                _authLabel != null
+                    ? _moduleContributor
+                    : throw new ViewNotBuiltException();
+            set {
+                if (_authLabel == null) throw new ViewNotBuiltException();
+
+                _moduleContributor = value;
+
+                _authLabel.Text             = $"{Strings.GameServices.ModulesService.ModuleManagement_AuthoredBy} {_moduleContributor.Name}";
+                _authLabel.BasicTooltipText = _moduleContributor.Username;
+            }
+        }
+
+        public string ModuleDescription {
+            get => _descLabel?.Text ?? throw new ViewNotBuiltException();
+            set => (_descLabel ?? throw new ViewNotBuiltException()).Text = value;
+        }
+
+        private IEnumerable<Version> _moduleVersions;
+
+        public IEnumerable<Version> ModuleVersions {
+            get =>
+                _versionDropdown != null
+                    ? _moduleVersions
+                    : throw new ViewNotBuiltException();
+            set {
+                if (_versionDropdown == null) throw new ViewNotBuiltException();
+
+                _moduleVersions = value;
+
+                _versionDropdown.Items.Clear();
+                _versionDropdown.Items.AddRange(CollectionUtils.Select(_moduleVersions, v => v.ToString()));
+            }
+        }
+
+        private Version _selectedVersion;
+
+        public Version SelectedVersion {
+            get =>
+                _versionDropdown != null
+                    ? _selectedVersion
+                    : throw new ViewNotBuiltException();
+            set {
+                if (_versionDropdown == null) throw new ViewNotBuiltException();
+
+                _selectedVersion = value;
+
+                _versionDropdown.SelectedItem = _selectedVersion.ToString();
+            }
+        }
+
+        private void SetRelationIcon(PkgVersionRelationship pkgVersionRelationship) {
+            switch (pkgVersionRelationship) {
+                case PkgVersionRelationship.NotInstalled:
+                    _statusImage.Visible      = false;
+                    break;
+                case PkgVersionRelationship.CanUpdate:
+                    _statusImage.Texture          = GameService.Content.GetTexture("common/157397");
+                    _statusImage.BasicTooltipText = "A newer version of this module is available.";
+                    _statusImage.Visible          = true;
+                    break;
+                case PkgVersionRelationship.CurrentVersion:
+                    _statusImage.Texture          = GameService.Content.GetTexture("common/157330");
+                    _statusImage.BasicTooltipText = "This module is installed and up to date.";
+                    _statusImage.Visible          = true;
+                    break;
```

Comment:
localization support is missing here too

Reply:
256177e Thanks!